modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryMvTableStorageTest.java

```diff
@@ -57,10 +60,18 @@ public class VolatilePageMemoryMvTableStorageTest extends AbstractMvTableStorage
 
     @BeforeEach
     void setUp(
+            TestInfo testInfo,
             @InjectConfiguration VolatilePageMemoryStorageEngineConfiguration engineConfig,
             @InjectConfiguration("mock.profiles.default = {engine = \"aimem\"}")
             StorageConfiguration storageConfiguration
     ) {
+        if (testInfo.getTags().contains(LOW_MEMORY_TAG)) {
```

Comment:
Where did this come from?

Reply:
By default we allocate 3 gigabytes of memory, so we need a lot of iterations to overflow memory if we have leakages. I.e. without my fix volatile memory fails only after 51 thousand iterations with default memory settings

Reply:
Removed. Moved test to volatile test class (at first didn't realize that it's not valid for persistent), and added more iterations - it still takes only 4 seconds, but without my fix overflows even with default memory settings